### PR TITLE
remove all data format #11

### LINKED ISSUE #11: remove all data format
remove all data format

### PR DESCRIPTION
**Pull request by darkarki83**

### CODE REVIEW

File: src/Net.Utils.CloudWatchHandler/Services/LogStreamService.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ public LogStreamService(IAmazonCloudWatchLogs client, LogStreamManager logStream
     }
 
     public static string GenerateLogStreamName(string? prefix)
-        => $"{prefix}-{DateTime.UtcNow:yyyy-MM-ddTHH:mm}";
+        => $"{prefix}-{DateTime.UtcNow:yyyy-MM-ddTHH-mm}";
```
Comment (Lomet) [inline src/Net.Utils.CloudWatchHandler/Services/LogStreamService.cs]: lets remove this

File: src/Net.Utils.CloudWatchHandler/Helpers/LogStreamManager.cs [hunk 2]
```diff
@@ -20,7 +20,7 @@ public virtual bool ShouldCreateNewStream(int streamCreationIntervalInMinutes)
         if (string.IsNullOrEmpty(CurrentLogStreamName))
             return true;
 
-        var timeSinceLastStreamInMinutes = (DateTime.UtcNow - LastStreamDate).TotalMinutes;
+        var timeSinceLastStreamInMinutes = (int)Math.Round((DateTime.UtcNow - LastStreamDate).TotalMinutes);
```
Comment (Lomet) [inline src/Net.Utils.CloudWatchHandler/Helpers/LogStreamManager.cs]: lets remove this

File: src/Net.Utils.CloudWatchHandler/Services/LoggingService.cs [hunk 3]
```diff
@@ -16,18 +17,18 @@ public LoggingService(IAmazonCloudWatchLogs cloudWatchClient, LogStreamService l
         _logStreamService = logStreamService ?? throw new ArgumentNullException(nameof(logStreamService));
     }
 
-    public virtual async Task LogMessageAsync(MessageData messageData)
+    public virtual async Task LogMessageAsync(LogConfiguration logConfiguration)
     {
         var logEvent = new InputLogEvent
         {
             Timestamp = DateTime.UtcNow,
```
Comment (Lomet) [inline src/Net.Utils.CloudWatchHandler/Services/LoggingService.cs]: lets remove this - log got his time

File: tests/Net.Utils.CloudWatchHandler.Tests/Models/LogConfigurationTests.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        logConfiguration.Should().NotBeNull();
+    }
+
+    [Fact]
+    public void GenerateLogStreamName_ShouldGenerateCorrectly_WhenPrefixIsProvided()
+    {
+        const string prefix = "testPrefix";
+
+        var generatedLogStreamName = LogStreamService.GenerateLogStreamName(prefix);
+        var expectedSuffix = DateTime.UtcNow.ToString("yyyy-MM-ddTHH-mm");
```
Comment (Lomet) [inline tests/Net.Utils.CloudWatchHandler.Tests/Models/LogConfigurationTests.cs]: this not part of the scope

Comment (darkarki83) [reply tests/Net.Utils.CloudWatchHandler.Tests/Models/LogConfigurationTests.cs]: task 1 done

File: src/Net.Utils.CloudWatchHandler/Services/LoggingService.cs (hunk 3 above)
Comment (darkarki83) [reply src/Net.Utils.CloudWatchHandler/Services/LoggingService.cs]: required CloudWatchLogs

File: src/Net.Utils.CloudWatchHandler/Services/LogStreamService.cs (hunk 1 above)
Comment (darkarki83) [reply src/Net.Utils.CloudWatchHandler/Services/LogStreamService.cs]: task 2 done

**Comment by darkarki83** [comment]
image

**Pull request merged**